packages/backend/src/core/entities/MetaEntityService.ts

```diff
@@ -66,7 +67,7 @@ export class MetaEntityService {
 			feedbackUrl: instance.feedbackUrl,
 			impressumUrl: instance.impressumUrl,
 			privacyPolicyUrl: instance.privacyPolicyUrl,
-			disableRegistration: instance.disableRegistration,
+			disableRegistration: envOption.disableRegistration ? envOption.disableRegistration : instance.disableRegistration,
```

Comment:
環境変数で指定した場合は強制的にその設定。
そうでない場合は従来通りコントロールパネルの設定が使用されるように。

Reply:
SignupApiService と同じくこれでよくない？
```suggestion
			disableRegistration: instance.disableRegistration || envOption.disableRegistration,
```